testing/helm/tests/env.yaml

```diff
@@ -38,6 +38,7 @@ dfuserver:
     
 eclagent:
 - name: eclagent
+  maxActive: 1
```

Comment:
Previously defaulted to 100 (!?).  Should discuss what the correct value is.

Reply:
as it's just a schema validation file, I thought 1 (min) was fine for this test.

Shall I change to match defaults?

Reply:
no, I didn't spot they were validation files - they're fine as they are.